src/Service/PackageSynchronizer/ComposerPackageSynchronizer.php

```diff
@@ -54,7 +56,16 @@ public function synchronize(Package $package): void
                 }
             }
 
+            if (!$latest) {
+                throw new \RuntimeException('Package not found');
+            }
```

Comment:
this can be check earlier, right after `$packages = $repository->getPackages()`, for example:
```php
if($packages === []) {
}
```